extensions/wikia/Search/js/WikiaSearch.js

```diff
@@ -36,10 +76,18 @@ require(['search-tracking', 'uuid', 'wikia.trackingOptIn'], function(searchTrack
 				}
 			});
 
-			$('.result-link').on('click', function(event) {
+			$('.result h1 .result-link').on('click', function(event) {
```

Comment:
The selector seems a bit fragile. How about we use a different class for the community link?

Let's also make sure how is this code related to `app/skins/oasis/js/Tracking.js`.